README.md

```diff
@@ -55,29 +55,55 @@ The criteria for picking these event names included:
+
+1. Don't assume any specific execution order
+2. Assume the list of suites and tests provided in runStart is in source order, and the output should honour that order.
+
+For more background check [#62](https://github.com/js-reporters/js-reporters/issues/62).
+
 ### Event Data
 
-Based on the discussion in [#12](https://github.com/js-reporters/js-reporters/issues/12#issuecomment-120483356), there are two basic data structures: Suites and Tests. A test represents an atomic test/spec/`it()`. A suite contains tests and optionally other suites. This can represent both flat structures like used by QUnit as well as nested suites like used by Jasmine or Mocha. The data structures are defined as follows:
-
- - **Test**: A test holds basic information on a single test/spec. It has the following set of required attributes:
-   - `testName`: name of the test
-   - `suiteName`: name of the suite the test belongs to
-   - `status`: result of the test. Can `passed`, `failed` or `skipped`. A skipped test is disabled, i.e. it will not be executed.
-   - `runtime`: execution time in milliseconds
-   - `errors`: array containing all errors. Depending on the framework, this is a single exception or a list of failed assertions. Will be empty for statuses other than failed.
- - **Suite**: A suite is a collection of tests and potentially other suites.
-   - `name`: name of the suite
-   - `childSuites`: array with all direct subsuites
-   - `tests`: array containing all tests that directly belong to the suite (but not to a child suite)
-   - `runtime`: execution time of the whole suite in milliseconds (including child suites)
-   - `status`: summarized status of the suite
-      - `failed`, if at least one test failed
-      - `skipped`, if all tests in the suite are skipped (and there is at least one skipped test)
-      - `passed`, if there is at least one passed test and all other tests are skipped or if there are no tests in the suite.
+Based on the discussion in [#79](https://github.com/js-reporters/js-reporters/issues/79), js-reporters establishes also a minimum set of properties for the emitted assertions, failed or passed:
+
+- **Assertion**: `Object` - An object which contains information of a single assertion/expectation.
+ - **passed**: `Boolean` - `true` for a passed assertion, `failed` for a failed assertion
+ - **actual**: `*` - the actual value passed to the assertion, should coincide with `expected` for passed assertions
+ - **expected**: `*` - the expected value passed to the assertion, should coincide with `actual` for passed assertions
+ - **message**: `String` - a description
+ - **stack**: `String|undefined` - represents the stack trace for a failed assertion, for a `passed` one it is `undefined`
+
+The content of the properties will vary based on the used assertion library and of course assertion libraries can extend with other properties.
```

Comment:
This is unclear to me. What exactly can vary? The properties defined here don't really seem dependent on the assertion library.

"of course" doesn't fit in here, let's drop that. Maybe use a separate sentence, something like "Assertion libraries can add additional properties (not defined here) to the assert object."